src/django/api/serializers/__init__.py

```diff
@@ -3,7 +3,7 @@
 from .contributor.contributor_list_query_params_serializer import (
     ContributorListQueryParamsSerializer
 )
-from .contributor.contributor_webhook_serializer import ContributorWebhookSerializer
+# from .contributor.contributor_webhook_serializer import ContributorWebhookSerializer
```

Comment:
Why did you comment it out? If it's a typo, please delete this line.

Reply:
removed